packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -54,10 +55,10 @@ class FramesBarPlotly {
 
   static List<Data> createFPSTraces() {
     final Data traceCpuGood = Data(
-      y: [],
-      x: [],
+      y: [0],
+      x: [-1],
```

Comment:
nit: comment why these default values are needed to work around strange plotly behavior.